roles/deploy/tasks/main.yml

```diff
@@ -0,0 +1,80 @@
+---
+- name: Initialize
+  deploy_helper: path="{{ project_root }}" state=present
+
+- name: Clone project files
+  git: repo="{{ project_git_repo }}"
+       dest="{{ project_source_path }}"
+       version="{{ project_version }}"
+       accept_hostkey=yes
+
+- name: write unfinished file
+  file: path="{{ project_source_path }}/{{ deploy_helper.unfinished_filename }}" state=touch
+
+- name: Copy files to new build dir
+  command: cp -pr {{ project_source_path }} {{ deploy_helper.new_release_path }}
```

Comment:
I understand that [f500/ansible-project_deploy_module](https://github.com/f500/ansible-project_deploy_module) is the proposed deploy_helper module for ansible-modules-extra. However, the tasks in question (Clone project files, Copy files to new build dir) come from [f500/ansible-project_deploy](https://github.com/f500/ansible-project_deploy).

I wasn't under the impression that this latter was also destined for ansible-modules-extra. But even if not, perhaps you'd like to maintain similarity. If in fact you're not concerned with similarity, you may want to prune a lot of unused stuff out of `/roles/deploy/defaults/main.yml` (which isn't part of 'deploy_helper').